assets/cart-notification.js

```diff
@@ -66,7 +66,7 @@ class CartNotification extends HTMLElement {
   handleBodyClick(evt) {
     const target = evt.target;
     if (target !== this.notification && !target.closest('cart-notification')) {
-      const disclosure = target.closest('details-disclosure');
+      const disclosure = target.closest('details-disclosure, header-menu');
```

Comment:
now that you've changed `details-disclosure` to `header-menu` I don't think you need to include it in here. Since from what I'm seeing this custom element isn't used anywhere now.

Reply:
Details-disclosure is extended by the ShareButton on the product page.

Reply:
Yeah it just feels a little odd to have some code for something that isn't used 🤔 

Maybe we could have an issue to remind ourselves to check/refactor in the future based on its use cases ?

Reply:
Yeah i agree, maybe there's a better way to target `details-disclosure` and all elements that extend it at once.